src/nethsec/firewall/__init__.py

```diff
@@ -8,11 +8,27 @@
 '''
 Firewall utilities
 '''
-
 import json
 import subprocess
+from enum import Enum
+
 from nethsec import utils
 
+PREFIX_CONFIG = 'ns_'
+
+
+class SpecialZones(Enum):
+    GUESTS = 'guests'
+    IPSEC = 'ipsec'
+    LAN = 'lan'
+    OPENVPN = 'openvpn'
+    RWOPENWRT = 'rwopenwrt'
+    RWWIREGUARD = 'rwwireguard'
+    TUNHOTSPOT = 'tunhotspot'
+    WAN = 'wan'
+    WIREGUARD = 'wireguard'
```

Comment:
Remove all unused enumerations.
Since here we are using only wan, I'd go with a simple string without enum